docs/export_results.rst

```diff
@@ -0,0 +1,180 @@
+Export Your Results
+===================
+Taichi has some functions that can help you to **export results to images or videos**. This tutorial will tell you how to do that step by step.
+Please make sure that your program runs well before reading this tutorial.
+
+Export Images
+-------------
+
+- There are 2 common ways to export the result of your program(presumably a Taichi tensor) to images. The first one working with ti.GUI that it is more friendly to begginers. 
+- The second approach is to call some built-in functions in Taichi to make things done, which is more flexible and harder.
+
+Export Images by ti.GUI
++++++++++++++++++++++++
+
+.. note::
+    
+    For beginners and debug purposes, we recommend this first method by ti.GUI!
+
+- The first method of exporting images is to use **ti.GUI.show()**. It can not only display the result to the screen but also export the result to your specified place.
+- About the format of saved images: **the type of image is fully determined by your suffix**. 
+- Though Taichi now supports exporting to .png, .jpg and .bmp these 3 mainstream formats, we strongly **recommend using .png** which has been tested well at this moment. For example:
+
+.. code-block:: python
+    
+    import taichi as ti
+    import os
+    pixel = ti.var(ti.u8, shape=(512, 512, 3))
+
+    @ti.kernel
+    def paint():
+        for I in ti.grouped(pixel):
+            pixel[I] = ti.random() * 255
```

Comment:
```suggestion
        for i, j, k in pixel:
            pixel[i, j, k] = ti.random() * 255
```
So that user don't confused by  `ti.grouped`, which is an advanced metaprogramming feature.